lib/tasks/taxonomy/export_taxonomy_to_json.rake

```diff
@@ -0,0 +1,59 @@
+require 'json'
+
+namespace :taxonomy do
+  desc <<-DESC
+    Exports an expanded taxonomy to a single JSON array
+  DESC
+  namespace :export do
+    task :json, [:root_taxon_id] => [:environment] do |_, args|
+      root_taxon_id = args.fetch(:root_taxon_id)
+      root_taxon = OpenStruct.new Services.publishing_api.get_content(root_taxon_id).to_h
+      taxonomy = Taxonomy::ExpandedTaxonomy.new(root_taxon.content_id)
+      taxonomy.build
+
+      flattened_taxonomy = flatten_taxonomy(taxonomy.child_expansion)
+
+      puts JSON.generate(flattened_taxonomy)
+    end
+
+    def flatten_taxonomy(taxon)
+      flattened_taxonomy = [convert_to_content_item(taxon)]
+
+      if taxon.children
+        taxon.children.each do |child_taxon|
+          flattened_taxonomy += flatten_taxonomy(child_taxon)
+        end
+      end
+
+      flattened_taxonomy
+    end
+
+    def convert_to_content_item(taxon, recursion_direction = nil)
```

Comment:
The service [BuildTaxon](https://github.com/alphagov/content-tagger/blob/master/app/services/taxonomy/build_taxon.rb) builds a taxon object from a payload. Would it make sense to have a `to_h` in the class `Taxon` method that would return the content item hash and call to it instead?